app-backend/db/src/main/scala/com/azavea/rf/database/Dao.scala

```diff
@@ -179,8 +179,15 @@ object Dao {
 
     // Filter to validate access to a specific object
     def authorize[M >: Model](user: User, objectType: ObjectType, objectId: UUID, actionType: ActionType)(implicit filterable: Filterable[M, Option[Fragment]]): QueryBuilder[Model] = {
+      val scenePublicF: Fragment = objectType match {
+        case ObjectType.Scene =>
+          fr"""
+          -- Match if scene is Public
+          visibility = 'PUBLIC' OR"""
```

Comment:
I think we'll need to add some other checks here...looks like as this is currently constructed that any user would be authorized to do any action with a public scene. We should limit this to `VIEW` and `DOWNLOAD` for now I think.